docs/src/assets/js/components-index.js

```diff
@@ -6,7 +6,7 @@
 
     if (matchMedia) {
         const mq = window.matchMedia("(max-width: 1023px)");
-        mq.addEventListener('change', WidthChange);
+        mq.addListener(WidthChange);
```

Comment:
```suggestion
        mq.addEventListener('change', WidthChange);
```

`addListener` is deprecated.